src/MPAGSCipher/ProcessCommandLine.cpp

```diff
@@ -27,6 +27,22 @@ bool processCommandLine(const std::vector<std::string>& cmdLineArgs,
             // Set the indicator and terminate the loop
             settings.versionRequested = true;
             break;
+        } else if (cmdLineArgs[i] == "--multi-cipher"){
+            // Handle taking multiple ciphers
+            // Next element will be the number of ciphers
+            if (i == nCmdLineArgs - 1) {
+                std::cerr << "[error] -multi-cipher requires a positive integer argument"
+                          << std::endl;
+                processStatus = false;
+                break;
+            } else {
+                nExpectedCiphers = std::stoul(cmdLineArgs[i + 1]);
+                std::cout << nExpectedCiphers << std::endl;
```

Comment:
Probably best to remove or comment out debug print outs like this when committing!